addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/automation.html

```diff
@@ -11,7 +11,7 @@ <H1>Automation Framework</H1>
 <p>
 It provides the following command line options:
 <ul>
-<li>-autorun &lt;filename&gt;		Run the automation jobs specified in the file.
+<li>-autorun &lt;source&gt;		Run the automation jobs specified in the file or from the URL.
```

Comment:
Align the descriptions?

Reply:
Being HTML it will not be aligned with white space so just left them as they were before.